x-pack/plugins/event_log/generated/mappings.json

```diff
@@ -267,6 +267,42 @@
                         }
                     }
                 },
+                "alert": {
```

Comment:
> I assume alert is being used here, instead of grouping under the peer alerting, to align with the alerts-as-data indices?

Yes, as we previously discussed, it would be nice to match the AAD namespace for better compatibility and to reduce cognitive overhead when switching between AAD documents and log events (no need to learn one more schema). You can find a list of existing AAD fields [here](https://docs.google.com/spreadsheets/d/1Cqv0bRx1BgY6VP9PY4FK6kppUEZJ5qMPoFgr5_XQXJI/edit#gid=1775019155). Currently, it doesn't contain the concrete fields we described in the RFC, as we're taking only the namespace from the AAD schema. But we could add the missing fields to the spreadsheet if needed.

That said, I'm okay with revisiting that decision. Let's discuss it further in the RFC.